lib/vtls/gtls.c

```diff
@@ -1029,7 +1029,8 @@ gtls_connect_step3(struct connectdata *conn,
   unsigned int verify_status = 0;
   gnutls_x509_crt_t x509_cert, x509_issuer;
   gnutls_datum_t issuerp;
-  char certbuf[256] = ""; /* big enough? */
+  gnutls_datum_t certfields;
+  char certname[64] = ""; /* limited to 64 chars by ASN.1 */
```

Comment:
We also get the name zero terminated, right? So doesn't this buffer need an extra byte for that?

Reply:
You're right, forgot that

Reply:
5556cddc27416bc52684d8656ce9193dcc40f7d3